src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write code here
+const columnTable = document.querySelector('.append-column');
+const rowTable = document.querySelector('.append-row');
+const columnRemoveTable = document.querySelector('.remove-column');
+const rowRemoveTable = document.querySelector('.remove-row');
+const table = document.querySelector('.field');
+
+columnTable.addEventListener('click', () => {
+  const rows = document.querySelectorAll('tr');
+
+  rows.forEach((x) => {
+    x.appendChild(document.createElement('td'));
+
+    if (x.children.length === 10) {
+      columnTable.disabled = true;
+    }
+
+    if (x.children.length > 2) {
+      columnRemoveTable.disabled = false;
+    }
+  });
+});
+
+rowTable.addEventListener('click', () => {
+  table.appendChild(document.querySelector('tr').cloneNode(true));
```

Comment:
This line clones the first row of the table to append a new row. If the first row has specific content or attributes, they will be duplicated. Consider creating a new row with the desired structure instead of cloning.